examples/project_dagster_university_start/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,11 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"build": {
+		"dockerfile": "Dockerfile",
+		"context": ".."
+	},
+	"forwardPorts": [
```

Comment:
In case it is useful, there is a cool way to surface ports in VS Code by adding [port attributes](https://containers.dev/implementors/json_reference/#port-attributes).

```json
"portsAttributes": {
        "3000": {
            "label": "Dagster"
        }
    }
```

The UI will show Dagster instead of `localhost` or `127.0.0.1`!

Reply:
thanks @davidgasquez ! https://github.com/dagster-io/dagster/pull/17187